### ISSUE TITLE
@sentry/cli/sentry-cli ENOENT

### ISSUE THREAD

**Issue by Valar103769** [getsentry/sentry-webpack-plugin#359]
### Environment

sentry.io

Which SDK and version?
```js
    "@sentry/tracing": "^6.18.2",
    "@sentry/vue": "^6.18.2",
```
 @file  vue.config.js
```js
const SentryWebpackPlugin = require('@sentry/webpack-plugin');

module.exports = {
  // 选项
  configureWebpack: {
    plugins: [
      new SentryWebpackPlugin({
        debug: true,
        authToken:'xxx',
        org: 'zuzhi',
        project: 'dc',
        // include: './dist',
        // configFile: 'sentry.properties',
        release: process.env.SENTRY_RELEASE
        // cleanArtifacts: true
      })
    ]
  }
};
```
`vue-cli-service serve`
 error  

Sentry CLI Plugin: spawn /Users/vayne/a/node_modules/@sentry/cli/sentry-cli ENOENT

**Comment by kamilogorek** [comment]
This is problem with installing `sentry-cli` binary. Based on your snippet, you are based in China, which sometimes have some issues with our Fastly CDN. Take a look how to set different repository source here - https://github.com/getsentry/sentry-cli/#node

tl;dr - set `sentrycli_cdnurl=https://github.com/getsentry/sentry-cli/releases/download/` in your `.npmrc` or run install as `SENTRYCLI_CDNURL=https://github.com/getsentry/sentry-cli/releases/download/ npm install`
